webapp/src/main/scala/io/iohk/atala/mediator/App.scala

```diff
@@ -1,4 +1,4 @@
-package io.iohk.atala.mediator
```

Comment:
https://github.com/input-output-hk/atala-prism-mediator/tree/rename_to_identus/webapp 
the folder ins still atala

Reply:
Thx I updated this